### DISCUSSION TITLE
How to install firefly on Azure?

### DISCUSSION THREAD

**Discussion by kkosciolek** [firefly-iii/firefly-iii#7675] [Firefly III]
Hi,
How can I install firefly on Azure?

**Comment by JC5** [comment]
Depends. You could just start a VM and run the manual?

**Reply by kkosciolek** [reply]
I would like to run it on azure web app.
I try "install" firefly on azure by multi-container (by example https://learn.microsoft.com/en-us/azure/app-service/quickstart-multi-container) but I have problems with mariadb.
